widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -155,6 +159,36 @@ public synchronized static void init(GliaWidgetsConfig gliaWidgetsConfig) {
         Dependencies.getGliaThemeManager().applyJsonConfig(gliaWidgetsConfig.uiJsonRemoteConfig);
     }
 
+    /**
+     * Retrieves an instance of {@link EngagementLauncher}.
+     *
+     * @param queueIds A list of queue IDs to be used for the engagement launcher.
+     *                 When null, the default queues will be used.
+     * @return An instance of {@link EngagementLauncher}.
+     * @throws GliaException with the {@link GliaException.Cause#INVALID_INPUT} if the SDK is not initialized.
+     */
+    @NonNull
+    public synchronized static EngagementLauncher getEngagementLauncher(@Nullable List<String> queueIds) {
+        Dependencies.setupQueues(queueIds);
+
+        return Dependencies.getEngagementLauncher();
+    }
+
+    /**
+     * Retrieves an instance of {@link EntryWidget}.
+     *
+     * @param queueIds A list of queue IDs to be used for the entry widget.
+     *                 When null, the default queues will be used.
+     * @return An instance of {@link EntryWidget}.
+     * @throws GliaException with the {@link GliaException.Cause#INVALID_INPUT} if the SDK is not initialized.
+     */
+    @NonNull
+    public synchronized static EntryWidget getEntryWidget(@Nullable List<String> queueIds) {
+        Dependencies.setupQueues(queueIds);
```

Comment:
I think it's more logical to implement it like this: `Dependencies.getConfigurationManager.setQueueIds(queueIds)`
I think so mainly because `setupQueues` is related to `ConfigurationManager` class, not `Dependencies`.